File: challenges/bubbleSort/solutions/bubblesort.js
var a = [5, 1, 4, 2, 8];
function bubblesort (a) {
	let swap = true;
	
	while(swap) {
		swap = false;
		for ( let i = 0; i < a.length;  i++) {
			if ( a[i] > a[i+1]) {
				[a[i], a[i+1]] = [a[i+1], a[i]];
				swap = true;
				console.log(swap);
				console.log(a);
			}
		}	
	}
	
	return a
}

console.log(a)

bubblesort(a)

File: challenges/bubbleSort/solutions/bubbleSort.rb
def bubble_sort(str: raise)
	sorted = false
	until sorted
		sorted = true
		str.size.times do |n|
			next if n == 0
			# swap if prior number is more than the first num
			if str[n-1] > str[n] 
				str[n-1], str[n] = str[n], str[n-1]  
				sorted = false
			end
		end
	end
	str
end


print bubble_sort(str: "zjbbca")